Setup Your Mac/Setup-Your-Mac-via-Dialog.bash

```diff
@@ -538,7 +538,10 @@ dialogSetupYourMacCMD="$dialogBinary \
 setupYourMacPolicyArrayIconPrefixUrl="https://ics.services.jamfcloud.com/icon/hash_"
 
 # shellcheck disable=SC1112 # use literal slanted single quotes for typographic reasons
-policy_array=('
+# If you would prefer to get your policyJSON externally replace it with:
+# policyJSON="$(cat /path/to/file.json)" # For getting from a file, replacing /path/to/file.json with the path to your file, or
+# policyJSON="$(curl -sL https://server.name/jsonquery)" # For a URL, replacing https://server.name/jsonquery with the URL of your file.
```

Comment:
Move "external file" instructions to larger comment block up above.

Reply:
Updated in [1.8.0-rc3](https://github.com/dan-snelson/dialog-scripts/blob/development/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L535-L537).